packages/plugin-session-replay-browser/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@amplitude/plugin-session-replay-browser",
-  "version": "0.7.1",
+  "version": "0.8.0",
```

Comment:
I _think_ that this will work, but not 100% how it will play with lerna, I have been relying on the CI for versioning for too long. Do you know @kevinpagtakhan?

Another option would be to create a `feat: update to new version` PR and update the README or another non-functional code in the package.